lang/src/test/kotlin/org/partiql/lang/eval/EvaluatingCompilerFromSourceByTests.kt

```diff
@@ -49,26 +49,26 @@ class EvaluatingCompilerFromSourceByTests : EvaluatorTestBase() {
     fun rangeOverListWithBy() = runEvaluatorTestCase(
         "SELECT VALUE addr FROM someList BY addr",
         session,
-        """[101, 102, 103]"""
+        "\$bag::[101,102,103]"
```

Comment:
I see some new tests use `$BAG_ANNOTATION` while others use `\$bag`.  Is there a reason for choosing one vs another in the specific cases?

Otherwise, since this is under your control within the PR, consistency would be better.